src/besseli.jl

```diff
@@ -193,3 +192,48 @@ function besseli_large_orders_scaled(v, x::T) where T <: Union{Float32, Float64,
 
     return T(coef*Uk_poly_In(p, v, p2, T))
 end
+
+function _besseli_continued_fractions(nu, x::T) where T
+    S = promote_type(T, Float64)
+    xx = S(x)
+    knu, knum1 = up_recurrence(xx, besselk0(xx), besselk1(xx), nu)
+    return 1 / (x * (knum1 + knu / steed(nu, x)))
+end
+
+function steed(n, x::T) where T
+    MaxIter = 1000
+    xinv = inv(x)
+    xinv2 = 2 * xinv
+    d = x / (n + n)
+    a = d
+    h = a
+    b = muladd(2, n, 2) * xinv
+    for _ in 1:MaxIter
+        d = inv(b + d)
+        a = muladd(b, d, -1) * a
+        h = h + a
+        b = b + xinv2
+        abs(a / h) <= eps(T) && break
+    end
+    return h
+end
+
+function besseli_large_argument(v, z::T) where T
+    MaxIter = 1000
+    S = promote_type(T, Float64)
+    z = S(z)
+    coef = exp(z) / sqrt(2*S(pi)*z)
+    fv2 = 4v^2
+    term = one(S)
+    res = term
+    s = -term
+    for i in 1:MaxIter
```

Comment:
It might make sense to make `i` be of type `T`. Not sure if it will change anything, but it might improve performance.

Reply:
This does slightly improve performance.